rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -383,7 +383,12 @@ QString game_list_frame::GetPlayTimeBySerial(const QString& serial)
 	const qint64 hours_played    = elapsed_seconds / 3600;
 	const qint64 minutes_played  = (elapsed_seconds % 3600) / 60;
 	const qint64 seconds_played  = (elapsed_seconds % 3600) % 60;
-
+	
+	// For anyone who was wondering why there need to be so many cases:
```

Comment:
*needs
:^)

Reply:
lol no

Reply:
"For anyone who was wondering why so many cases are needed:"? Kappa